source/collimation.f90

```diff
@@ -1743,34 +1747,21 @@ subroutine collimate_start_sample(nsample)
     ejv(1:napx00)   = myp(1:napx00)
   end if
 
-  do i = 1, napx00
-    ! FOR NOT FAST TRACKING ONLY
-    ejfv(j)=sqrt(ejv(j)**2-nucm(j)**2)
-    rvv(j)=(ejv(j)*e0f)/(e0*ejfv(j))
-    dpsv(j)=(ejfv(j)*(nucm0/nucm(j))-e0f)/e0f
-    oidpsv(j)=one/(one+dpsv(j))
-    moidpsv(j)=mtc(j)/(one+dpsv(j))
-    omoidpsv(j)=c1e3*((one-mtc(j))*oidpsv(j))
-    dpsv1(j)=(dpsv(j)*c1e3)*oidpsv(j)
-
-    partID(i)=i
-    parentID(i)=i
+  call part_updatePartEnergy(1)
 
-    do ieff =1, numeff
+  do i=1,napx00
```

Comment:
Unneeded variable, but apparently removed in a later PR.

Reply:
Huh? It's not unneeded, but there is no difference between `napx` and `napx00`, so `napx00` can be replaced.